src/read/op.rs

```diff
@@ -705,7 +705,11 @@ where
             }
             constants::DW_OP_stack_value => Ok(Operation::StackValue),
             constants::DW_OP_implicit_pointer | constants::DW_OP_GNU_implicit_pointer => {
-                let value = bytes.read_offset(encoding.format)?;
+                let value = if encoding.version != 2 {
+                    bytes.read_offset(encoding.format)?
+                } else {
+                    Offset::from_u64(bytes.read_address(encoding.address_size)?)?
```

Comment:
```suggestion
                    bytes.read_address(encoding.address_size).and_then(R::Offset::from_u64)?
```
Just for consistency with how the rest of the code is written.

Reply:
Adjusted for consistency.